dist/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Unamed Game</title>
+    </head>
+
+    <body>
+        <div id="content"></div>
+        <script src="main.js"></script>
+    </body>
+</html>
```

Comment:
Not that important, but github shows the message "no newline at end of file" for some other files as well. I think we don't get problems because of that but it is good practice to put this empty line at the end of a file.

Reply:
Everything else looks good to me! Prettier would be nice as well.

Reply:
Oh one more thing about css: css-file into dist? Or do we want to load it with webpack? I don't care. Maybe also a css-reset-file would be good.

Reply:
Another thing (I think it is the last :wink:): could you add also `devtool: 'source-map'` to webpack.config.js? I find it very helpful for debugging code. [(Here an example how you add it)](https://github.com/DavidMatalik/todo/blob/main/webpack.config.js)

Reply:
yep, with prettier installed this should be automatically fixed every time

Reply:
True =)